mock/server/authentication.go

```diff
@@ -40,6 +41,8 @@ const (
     "actor_scm_id",
     "commands",
     "image",
+	"image_name",
+	"image_tag",
```

Comment:
looks like these are not indented right?
![image](https://github.com/user-attachments/assets/e0529ecf-7ee4-4b06-9550-0451bbc61c1f)

Reply:
This should be resolved now.